packages/__docs__/buildScripts/ai-accessible-documentation/summaries-for-llms-file.json

```diff
@@ -0,0 +1,475 @@
+  },
+  {
+    "title": "RadioInput",
+    "summary": "Custom styled radio button component. Typically used within RadioInputGroup for proper grouping and behavior. Supports different sizes and should always be used in groups rather than individually."
+  },
+  {
+    "title": "RadioInputGroup",
+    "summary": "Manages groups of radio buttons with shared name attribute. Supports various layouts (default, inline, columns), toggle variant, disabled states, and accessibility. Handles selection state and change events for the entire group."
+  },
+  {
+    "title": "RangeInput",
+    "summary": "An HTML5 range input/slider component with customizable size and thumb variants. Supports React class and functional component implementations with interactive examples for real-time updates."
+  },
+  {
+    "title": "Rating",
+    "summary": "Displays 3- or 5-star ratings based on valueNow and valueMax props. Supports rounding decimals, customizable sizes, animations, and accessibility-compliant text formatting for screen readers."
+  },
+  {
+    "title": "Responsive",
+    "summary": "Renders components differently based on element or viewport size. Supports breakpoint-based props and conditional rendering. Use sparingly to avoid performance issues."
+  },
+  {
+    "title": "SVGIcon",
+    "summary": "Renders accessible inline SVG icons. Supports custom sizing, color theming, rotation, and display properties. Accepts SVG content via children or external source strings."
+  },
+  {
+    "title": "ScreenReaderContent",
+    "summary": "Renders content accessible only to screen readers, not visible on screen. Essential for providing hidden context or instructions to assistive technologies."
+  },
+  {
+    "title": "Search",
+    "summary": "Provides search input patterns including auto-activated, activated, and autocomplete search. Includes guidelines, accessibility support, and clear state management examples."
+  },
+  {
+    "title": "Select",
+    "summary": "Accessible combobox component for single or multiple selections. Supports autocomplete, grouping, async loading, icons, and screen reader announcements. Fully controllable with extensive state management examples."
+  },
+  {
+    "title": "Selectable",
+    "summary": "Low-level utility for building custom combobox widgets. Provides ARIA-compliant prop getters and state management hooks. Use existing Select component when possible."
+  },
+  {
+    "title": "SideNavBar",
+    "summary": "Experimental top-level navigation component with toggleable minimized/expanded states. Supports icons, Avatars, badges, and theming. API may change significantly."
+  },
+  {
+    "title": "SimpleSelect",
+    "summary": "Simplified Select component mimicking native <select> behavior. Supports uncontrolled/controlled usage, option groups, and icons. Less configurable than Select but easier to implement."
```

Comment:
Marekdown viewers might do silly things if you don't escape raw HTML tags, put this Select into backticks, e.g. 
```
`<select>`
```